lib/players/human.rb

```diff
@@ -0,0 +1,24 @@
+module Players
+  class Human < Player
+
+    attr_accessor:input
+
+    def user_input
```

Comment:
I put these methods here to pass the tests
( Players::Human
  inherits from Player
  #move
    asks the user for input and returns it
)